alerter/IThresholdAlerter.java

```diff
@@ -0,0 +1,6 @@
+
+
+public interface IThresholdAlerter {
+
+    Integer networkAlertStub(float celsius) ;
```

Comment:
Interface is defined so that the production code is unaware whether the implementor is test stub or a production code.
Hence the methods API naming should not include stub. Whereas , the implementation class of this interface can have the stub name.